ext/gd/gd_image_object.h

```diff
@@ -0,0 +1,37 @@
+/*
```

Comment:
We don't do these kind of headers for specific objects, this should be in the `.c` file

Reply:
@KalleZ The type isn't specific to the gd.c file, there's also gd_ctx.c which needs access to it.